src/main/java/codeu/controller/RegisterServletTest.java

```diff
@@ -16,22 +17,30 @@
     private HttpServletRequest mockRequest;
     private PrintWriter mockPrintWriter;
     private HttpServletResponse mockResponse;
+    private RequestDispatcher mockRequestDispatcher;
 
     @Before
     public void setup() throws IOException{
         registerServlet = new RegisterServlet();
+
         mockRequest = Mockito.mock(HttpServletRequest.class);
-        mockPrintWriter = Mockito.mock(PrintWriter.class);
         mockResponse = Mockito.mock(HttpServletResponse.class);
+        mockRequestDispatcher= Mockito.mock(RequestDispatcher.class);
+
+        Mockito.when(mockRequest.getRequestDispatcher("/WEB-INF/view/register.jsp"))
```

Comment:
import static org.mockito.Mockito.when; so that you don't have to keep saying Mockito.when.  There's this trend -- not sure I like it -- of making things "fluent" so that it reads kind of like badly-written english: "when(thing.getOther()).thenReturn(thisFakeThing)".